source/onboard/migrate-from-slack.rst

```diff
@@ -0,0 +1,191 @@
+--------------------------
+
+Slack offers two ways to `export your data from their product <https://get.slack.help/hc/en-us/articles/201658943-Export-your-workspace-data>`_.
+
+1. Regular export - Contains only public channel posts. This does not include private channels, DMs, or group conversations. This can be generated from **Slack > Administration > Workspace settings > Import/Export Data > Export > Start Export**.
+2. Corporate export - Contains all posts. This includes public channels, private channels, DMs and group messages. You must `request this export type from Slack directly <https://slack.com/help/articles/1500001548241-Request-to-export-all-conversations>`__.
+
+You will receive a zip file containing the following contents:
+
+- Channels (``channels.json``)
+- Users (``users.json``)
+- Direct messages (``dms.json``) (Corporate export)
+- Private channels (``groups.json``) (Corporate export)
+- Group direct messages (``mpims.json``) (Corporate export)
+- App activity logs (``integration_logs.json``)
+- Folders containing posts for every public channel
+- Folders containing posts for every private channel (Corporate exports)
+
+.. note::
+
+  - Refer to the `Slack help article <https://slack.com/help/articles/220556107-How-to-read-Slack-data-exports>`__ for additioanl details on zip file contents.
+  - As a proprietary SaaS service, Slack is able to change its export format quickly and without notice. If you encounter issues not mentioned in the following documentation, please let the Mattermost Product Team know by `filing an issue <https://handbook.mattermost.com/contributors/contributors/ways-to-contribute>`__.
+
+3. Download file attachments and email addresses
+------------------------------------------------
+
+When you download your Slack export zip file, some data will be missing from the zip file:
+- User emails
+- Uploaded attachment contents
+
+We need to create a Slack app in order to gather these contents. Follow these steps to create a Slack app:
+
+1. Go to https://api.slack.com/apps.
+2. Select **Create New App**.
+3. Select **From scratch**.
+4. Name the app **Slack Advanced Exporter** and select the workspace. You'll have to do this for every workspace. Then select **Create App**.
+5. Select **OAuth & Permissions** on the left-hand side of the screen. Then scroll down to **Scopes**.
+6. Under **Bot Token Scopes** type in and select the following scopes:
+  - ``users:read``
+  - ``users:read.email``
+
+7. Scroll up and select **Install to Workspace**.
+8. Grant the app permissions when prompted.
+9. Copy the **Bot User OAuth Token** and save it somewhere convenient.
+
+We'll now use **Bot User OAuth Token** with the ``slack-advanced-exporter`` tool to download emails and attachments. Download the latest release of ``slack-advanced-exporter`` for your OS and architecture `here <https://github.com/grundleborg/slack-advanced-exporter/releases/>`__ and extract the executable from the download.
```

Comment:
Should we make a note that the `slack-advanced-exporter` is not a Mattermost _product_? (Because it isn't, right?)

Reply:
Transparency here will clarify ownership and build trust. Yes from me!

Reply:
@fmartingr I would say that it is a Mattermost product, as it was written by a Mattermost employee at the time. What are your thoughts on getting this moved to the Mattermost GitHub org, as an alternative to noting that it is not a Mattermost product?

Reply:
I'm ok with moving it to the org if we plan to actively support it, but I can't make that decision since I don't know which team would own it and if they have bandwidth for it.